package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "app",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
This file is not needed.

Reply:
As stated, the code is rolled back to a previous version.